src/api/Auth/index.ts

```diff
@@ -78,6 +83,22 @@ export class AuthAPI extends Request {
     return response;
   }
 
+  /**
+   * Create the current user (in order to setup authentication).
+   */
+  public async createCurrentUser(this: Request<User>, params: ICreateCurrentUserParams) {
```

Comment:
can we name it createFirstUser ? just a suggestion

Reply:
I renamed it `setupAuthentication()` as agreed during our Zoom call.